getting-started-2.0.md

```diff
@@ -0,0 +1,155 @@
+1. Add a file called `footer.md` to your GitHub repository.
+1. Add some content you would like to be shown in the footer. Here's a [proposal](https://raw.githubusercontent.com/adobe/helix-example/master/footer.md "example footer.md").
+1. Click the _Commit changes_ button.
+1. Reload `https://<my-repo>-<my_org>.project-helix.page/` to see header and footer in action.
+   <br>![TODO: Helix Pages rendering your header and footer](./dummy.png)
+
+## Step 3: Add your own HTML files
+
+> You'll need:
+> - basic HTML knowledge
+
+If you have existing HTML files or snippets that you wish to reuse, you can simply add them to your GitHub repository and have _Helix_Pages_ merge and deliver them for you:
+
+1. Add a `static.html` file to your GitHub repository. The file name can be anything, just make sure it has a `.html` extension.
+1. Add your HTML. Maybe [like so](https://raw.githubusercontent.com/adobe/helix-example/master/htdocs/static.html "example HTML file")?
+1. Click _Commit changes_.
+1. Refresh `https://<my-repo>-<my_org>.project-helix.page/static.html` to see the rendered result.
+
+> **Note:** If there is a `.md` and `.html` with the same name, _Helix Pages_ will give precendence to the Markdown file. Rename or remove the Markdown file to have the HTML file rendered instead.
+
+> **Hint:** If you add a static `header.html` or `footer.html` instead of their Markdown versions, _Helix Pages_ will embed these for you.
+
+## Add your own styling
+
+> You'll need:
+> - basic CSS knowledge
+
+_Helix Pages_ allows you to override the default CSS. You can either do this directly in GitHub, or start developing locally.
+
+### GitHub
+
+1. Add a `style.css` file to your GitHub repository.
+1. Insert your CSS rules. See the [default CSS](https://raw.githubusercontent.com/adobe/helix-pages/master/htdocs/style.css "default CSS") for pointers where you can add overrides.
+1. Click _Commit changes_.
+1. Reload `https://<my-repo>-<my_org>.project-helix.page/` to view your style changes in action.
+1. Repeat steps 2 - 4 until you are happy with how your page looks.
+
+### Local development
+
+For an improved developer experience, using a local environment and the IDE of your choice, we proudly introduce you to the _Helix CLI_ (`hlx`). For the sake of simplicity, we just assume you are using _Visual Studio Code_ and have a `code` alias for it.
+
+> You'll need:
+> - a shell :)
+> - an IDE, e.g. [Visual Studio Code](https://code.visualstudio.com/)
+> - [Node.js](https://nodejs.org/en/) 8.9 or higher and `npm`
+> - [Helix CLI](https://www.npmjs.com/package/@adobe/helix-cli)
+> - [Git](https://git-scm.com/)
+
+1. Open your favorite shell.
+1. Change to the directory where you normally keep your code, e.g. type `cd ~/code`.
+1. Type `git clone https://github.com/<my-org>/<my-repo>.git`
+   <br>This will your GitHub repository locally into a folder with the same name as your GitHub repository.
+1. Type `cd <my_repo>`
+1. Type `hlx up &`
```

Comment:
why `&` ? (doesn't work on windows)

Reply:
Because it always happens to me that `hlx` is the front-most process and if I want to type `code .` next, I either have to open a new terminal tab or suspend and throw it in the background first :)

Open to a generic solution that would work for all OSs...

Reply:
but this is a problem of the user. if the user is savy, it will know to use `&` or a second terminal. If not, well...

you can also tell him to do:
```
ctrl-z
$ open .
$ fg
```

but this would also be very platform specific.

Reply:
Ok, you're right. I'll remove it.

Reply:
also, you'd need to explain how to stop the server....